README.md

```diff
@@ -44,15 +44,22 @@ Accessibility Insights for Android Service is a service for Android that helps i
 		* Ensure that the **app** is selected in the left navigation pane
 		* Under **General** > **Launch Options**, change **Launch** from "Default Activity" to "Nothing"
 		* Select **Apply**, then **Ok**
-    
-1. 	Run the app from Android Studio (this will install the APK onto the emulator). The app won't show up in the list of programs--it shows up as a downloaded accessibility service. It will be off by default. To activate it:
+
+1. Run the app from Android Studio (this will install the APK onto the emulator). The app won't show up in the list of programs--it shows up as a downloaded accessibility service. It will be off by default. To activate it:
     * Open the Settings app in the emulator. *If you have trouble opening the emulator, see [Unable to start emulator](#unable-to-start-emulator) under [Known Issues](#known-issues) below*
     * Scroll down and select **Accessibility**
     * Select "Accessibility Insights for Android Service" under **Downloaded services**
     * Toggle on **Use service** to enable the service
     * Select **Allow** on the resulting dialog to grant the necessary permissions
     * An "Exposing sensitive info during casting/recording" dialog should appear. Select **Start now** 
     
+#### Viewing raw JSON results in browser
+
+1. While the service is running, forward the emulated device port `62442` to the same port on your machine by running `adb forward tcp:62442 tcp:62442` from a command prompt.
```

Comment:
Two thoughts on this:
1. The HTTP get request is valid now, but it might change in the future (for example, we've discussed using gRPC, which has a separate protocol). If we document this, we might want to include some disclaimer text that this isn't guaranteed to work in future releases.
2. Part of the motivation of auto-selecting the port was to prevent the user from setting the port forwarding (and potentially conflicting with other forwarded ports). As such, it might make sense to just use whatever port forwarding the client has configured. To do this, just start the client, go to a command prompt, and type in `adb forward --list`. You'll get back something like this:
```
ce11182b4587381d0b7e tcp:62442 tcp:62442
```

The 1st number is the port for localhost, and the second number is the port on the device (and is always 62442)

Reply:
ooh that's handy!

Reply:
Is it auto-selected? I'm not very familiar with how the service works but from what I can tell it'll always be the number listed [here](https://github.com/microsoft/accessibility-insights-for-android-service/blob/f5679ec37f8cc1c1833c8a139586a6b8e219d838/AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/ServerThread.java#L11). I could link to this file perhaps?

Reply:
@madalynrose, the 2nd number will always be 62442--that's the port that is hardcoded into the service. The 1st number will _usually_ (but not always) be 62442. It's a port number that is selected at runtime--we start with 62442, then (if 62442 is in use), it tries 62443, then 62444, up to some upper bound (62542, I think).